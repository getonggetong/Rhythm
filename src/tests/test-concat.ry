/*	Concatentation Test

	Tests the following:
	1. Two notes into a sequence
	2. Note to end of a sequence
	3. Chord to end of sequence
*/

init() {

	print(A4 :: A4);
	print([A4,B4] :: C4);
	print([A4,B4] :: [[C4,D4]]);

}