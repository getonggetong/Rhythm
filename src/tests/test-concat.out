[A4,A4]
[A4,B4,C4]
[A4,B4,[C4,D4]]
